spring-integration-mail/src/main/java/org/springframework/integration/mail/config/MailReceiverFactoryBean.java

```diff
@@ -185,6 +195,9 @@ private MailReceiver createReceiver() {
 		}
 		receiver.setMaxFetchSize(this.maxFetchSize);
 		receiver.setSelectorExpression(selectorExpression);
+		if (this.userFlag != null) {
```

Comment:
Oh, I see, yes, `StringUtils.hasText()` - thanks; just in case someone sets it in Java. Please fix as necessary.